onnxruntime/core/providers/acl/math/gemm.h

```diff
@@ -8,6 +8,7 @@
 #include "core/util/math.h"
 #include "core/util/math_cpuonly.h"
 #include "core/providers/cpu/math/gemm.h"
+#include "core/providers/cpu/math/gemm_helper.h"
```

Comment:
So, this includes a header into another header which I feel may disseminate it too wide. Where exactly gemm_helper.h definitions are needed? My understanding they are not needed in gemm.h but some place else. We want to include headers into .cc files where they are required.

Reply:
it's needed for the Compute() method.
probably better to move the Compute() to .cc and include the headers from there.

Reply:
Should I make that change in this PR? I noticed you approved the current fix.

Reply:
we can refactor later. let's just fix the ACL build for now.